src/test/java/uk/gov/hmcts/ccd/domain/service/processor/FieldProcessorServiceTest.java

```diff
@@ -0,0 +1,238 @@
+package uk.gov.hmcts.ccd.domain.service.processor;
+
```

Comment:
we should add a good bunch of  integration tests to protect ourselves from regressions